### ISSUE TITLE
Edge cases

### ISSUE THREAD

**Issue by mintcd** [mintcd/annotation#5]

**Comment by mintcd** [comment]
https://www.ieltsadvantage.com/2023/01/15/ielts-writing-task-2-sample-essays/

It redirects right to the captcha page, which cannot be escaped.

**Comment by mintcd** [comment]
Glasp can clone the page perfectly. It seems we have to move script injection back to client> https://www.cambridge.org/core/elements/epistemology-of-logic/CDF2419972F044234448B54AC3EAF5CD?fbclid=Iwb21leANK-2hjbGNrA0r7SWV4dG4DYWVtAjExAAEeex_aHm1Mfxm5dfl8KrSVoESN2FfrEPlyChpdBbOQgKiDFAaFIvQpjuvOk0E_aem_7O5d8QMpctXnYpWllx2Hvg
> 
> Could not fetch the file named a11y-1.0.40.min.js?_=1763501559769 initiate by sentry.min.js. I cannot even find the file in the original page.
> 
> 
> Plan: do an intense debug by trying to split the files into lines.
